pkg/pluginmanager/manager_test.go

```diff
@@ -213,6 +213,26 @@ func Test_InstallStandalonePlugin(t *testing.T) {
 	assertions.Equal(1, len(installedPlugins))
 	assertions.Equal("login", installedPlugins[0].Name)
 
+	// Install login (standalone) plugin with just vMajor as version
+	err = InstallStandalonePlugin("login", "v0", configtypes.TargetUnknown)
+	assertions.Nil(err)
+	// Verify installed plugin
+	installedPlugins, err = pluginsupplier.GetInstalledPlugins()
+	assertions.Nil(err)
+	assertions.Equal(1, len(installedPlugins))
+	assertions.Equal("login", installedPlugins[0].Name)
+	assertions.Equal("v0.2.0", installedPlugins[0].Version)
+
+	// Install login (standalone) plugin with just vMajor.Minor as version
+	err = InstallStandalonePlugin("login", "v0.2", configtypes.TargetUnknown)
+	assertions.Nil(err)
+	// Verify installed plugin
+	installedPlugins, err = pluginsupplier.GetInstalledPlugins()
+	assertions.Nil(err)
+	assertions.Equal(1, len(installedPlugins))
+	assertions.Equal("login", installedPlugins[0].Name)
+	assertions.Equal("v0.2.0", installedPlugins[0].Version)
+
```

Comment:
Would it be possible to add a login plugin of version `v0.20.0` in the DB (in `manager_helper_test.go`) to make sure that installing `v0.2` will not match the newer `v0.20.0`

Reply:
Yup. Meant to do that earlier as well. Just added some tests. Thanks